File: ttp-composite/linux/go/coverage/metadata_ec2/main.go
package main

import (
	"fmt"
	"github.com/michaellong/telemetry-lab/ttp-composite/linux/go/coverage/internal/fixture"
	"github.com/michaellong/telemetry-lab/ttp-composite/linux/go/coverage/internal/network"
)

func main() {
	if !fixture.Begin("metadata_ec2") {
		return
	}
	network.Metadata()
	fmt.Print("CASE_OK metadata_ec2\n")
}

File: ttp-composite/linux/cpp/coverage/metadata_ec2/main.cpp
#include "../network.hpp"

int main(int argc, char **argv) {
    if (!fixture_begin(argc,argv,"metadata_ec2")) return 0;
    tcp_exchange("169.254.169.254",0);
    std::cout << "CASE_OK metadata_ec2\n";
    return 0;
}
